src/caa-cases/containers/caa-cases/caa-cases.component.ts

```diff
@@ -145,29 +176,31 @@ export class CaaCasesComponent implements OnInit {
   }
 
   public loadDataFromStore(): void {
-    if (this.caaCasesPageType === CaaCasesPageType.UnassignedCases) {
-      this.store.dispatch(new fromStore.LoadUnassignedCases({
-        caseType: this.currentCaseType,
-        pageNo: this.currentPageNo,
-        pageSize: this.paginationPageSize,
-        caaCasesFilterType: this.selectedFilterType,
-        caaCasesFilterValue: this.selectedFilterValue
-      }));
-      this.cases$ = this.store.pipe(select(fromStore.getAllUnassignedCaseData));
-    } else {
-      this.store.dispatch(new fromStore.LoadAssignedCases({
-        caseType: this.currentCaseType,
-        pageNo: this.currentPageNo,
-        pageSize: this.paginationPageSize,
-        caaCasesFilterType: this.selectedFilterType,
-        caaCasesFilterValue: this.selectedFilterValue
-      }));
-      this.cases$ = this.store.pipe(select(fromStore.getAllAssignedCaseData));
+    if (this.currentCaseType) {
```

Comment:
@johnbenjamin-hmcts - would be good to have this covered by a unit test, even though there wasn't one previously. Something basic, just to check if actions are being dispatched when they should, and not when they shouldn't

Reply:
Thank you @Daniel-Lam I have now written couple of unit tests to cover this.

Reply:
Thanks for adding some unit tests, @johnbenjamin-hmcts.